src/main/java/com/epam/aidial/cfg/service/export/ConfigExportServiceSecuredImpl.java

```diff
@@ -60,4 +71,20 @@ private CoreModel mapModel(CoreModel value) {
         return model;
     }
 
+    private CoreToolSet mapToolSet(CoreToolSet value) {
+        CoreResourceAuthSettings authSettings = new CoreResourceAuthSettings();
+        authSettings.setClientSecret(StringUtils.EMPTY);
+
+        if (value != null
```

Comment:
can we skip toolsets without clientSecret in secretConfig?

Reply:
Done